src/components/overlay/pages/overlay-interactive.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import DocsActiveBlock from 'components/DocsActiveBlock';
+import Overlay from '../Overlay';
+
+const Overlays = () => {
+  const settings = [];
+
+  return <div>
+    <DocsActiveBlock settings={settings}>
+      <div>
+        <Overlay/>
```

Comment:
I checked in other projects and we also use it in the way you described (wrapping content).